From b46e3ffb6a789286b6ed69acf3b362bc8bbfeb59 Mon Sep 17 00:00:00 2001
From: "Wood, Brian J" <brian.j.wood@intel.com>
Date: Thu, 4 Apr 2019 10:44:38 -0700
Subject: [PATCH 1341/1374] drm/i915: Remove unused function causing GCC
 compilation warning

This commit is removing the unused function update_primary_plane_state()
that when compiling with the GCC option -Werror=unused-function
causes build failures.


Change-Id: Ib0d98381c7457f79c3a8aa14fac65588951b3d7d
Signed-off-by: Wood, Brian J <brian.j.wood@intel.com>
---
 drivers/gpu/drm/i915/intel_initial_modeset.c | 27 ---------------------------
 1 file changed, 27 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_initial_modeset.c b/drivers/gpu/drm/i915/intel_initial_modeset.c
index 74630533..222c6b6 100644
--- a/drivers/gpu/drm/i915/intel_initial_modeset.c
+++ b/drivers/gpu/drm/i915/intel_initial_modeset.c
@@ -208,33 +208,6 @@ static int update_connector_state(struct drm_atomic_state *state,
 	return 0;
 }
 
-static int update_primary_plane_state(struct drm_atomic_state *state,
-				      struct drm_crtc *crtc,
-				      struct drm_display_mode *mode,
-				      struct drm_framebuffer *fb)
-{
-	int hdisplay, vdisplay;
-	struct drm_plane_state *primary_state;
-	int ret;
-
-	primary_state = drm_atomic_get_plane_state(state, crtc->primary);
-	ret = drm_atomic_set_crtc_for_plane(primary_state, crtc);
-	if (ret)
-		return ret;
-	drm_mode_get_hv_timing(mode, &hdisplay, &vdisplay);
-	drm_atomic_set_fb_for_plane(primary_state, fb);
-	primary_state->crtc_x = 0;
-	primary_state->crtc_y = 0;
-	primary_state->crtc_w = hdisplay;
-	primary_state->crtc_h = vdisplay;
-	primary_state->src_x = 0 << 16;
-	primary_state->src_y = 0 << 16;
-	primary_state->src_w = hdisplay << 16;
-	primary_state->src_h = vdisplay << 16;
-	primary_state->rotation = DRM_MODE_ROTATE_0;
-
-	return 0;
-}
 
 static int update_atomic_state(struct drm_device *dev,
 			       struct drm_atomic_state *state,
-- 
2.7.4

